ibverbs/src/lib.rs

```diff
@@ -1046,6 +1072,36 @@ impl AsMut<ffi::ibv_gid> for Gid {
     }
 }
 
+/// A Global identifier entry for ibv.
+///
+/// This struct acts as a rust wrapper for `ffi::ibv_gid_entry`. We use it instead of
+/// `ffi::ibv_gid_entry` because `ffi::ibv_gid` is wrapped by `Gid`.
+pub struct GidEntry {
+    /// The GID entry.
+    pub gid: Gid,
+    /// The GID table index of this entry.
+    pub gid_index: u32,
+    /// The port number that this GID belongs to.
+    pub port_num: u32,
+    /// enum ibv_gid_type, can be one of IBV_GID_TYPE_IB, IBV_GID_TYPE_ROCE_V1 or IBV_GID_TYPE_ROCE_V2.
+    pub gid_type: ffi::ibv_gid_type,
+    /// The interface index of the net device associated with this GID.
+    /// It is 0 if there is no net device associated with it.
```

Comment:
```suggestion
    /// The interface index of the net device associated with this GID.
    ///
    /// It is 0 if there is no net device associated with it.
```
The empty line is needed for this to render correctly in rustdoc :)